pkg/api/controller.go

```diff
@@ -420,6 +427,74 @@ func compareImageStore(root1, root2 string) bool {
 	return isSameFile
 }
 
+type abstractStorageDriver struct {
```

Comment:
Let's try and **not** use the word "abstract" here

Reply:
Got rid of the type entirely, used config.StorageConfig instead